src/main/java/gov/cms/mat/cql_elm_translation/service/DataCriteriaService.java

```diff
@@ -208,9 +208,11 @@ private String splitByPipeAndGetLast(String criteria) {
     return parts[parts.length - 1];
   }
 
-  private String buildCriteriaType(String dataType) {
+  String buildCriteriaType(String dataType) {
     // e.g "Encounter, Performed" becomes "EncounterPerformed",
     // e.g for negation: "Assessment, Not Performed" becomes "AssessmentPerformed"
-    return dataType.replace(",", "").replace(" ", "").replace("Not", "");
+    return QdmDatatypeUtil.isValidNegation(dataType)
+        ? QdmDatatypeUtil.getTypeForNegation(dataType)
+        : dataType.replace(",", "").replace(" ", "").replace("Not", "");
```

Comment:
`.replace("Not", "")` not needed anymore?

Reply:
I left that there just in case we don't have a mapping for it